velox/expression/ExprToSubfieldFilter.cpp

```diff
@@ -404,6 +394,12 @@ std::unique_ptr<common::Filter> makeBetweenFilter(
             singleValue<int64_t>(lower), singleValue<int64_t>(upper));
       }
       return between(singleValue<int64_t>(lower), singleValue<int64_t>(upper));
+    case TypeKind::INTEGER:
```

Comment:
Should we put TypeKind::INTEGER above TypeKind::BIGINT to keep the natural order?